### PR TITLE
Added: Danish Translations

### PR DESCRIPTION
**Pull request by ManifestManah**
Added Danish translations and language support to the modification as a whole, I intend to maintain the Danish translations so if there's anything you find to be off or weird then let Pwned or I know and we'll see if we can find a more suitable translation for whatever you might find. (^.^)

### CODE REVIEW

**Comment by srpg** [comment]
ManifestManah it seems to remove some my finnish translations. 
https://github.com/ThaPwned/WCS/pull/34/commits/474a6f74e2fcf110ee0ad3e037ccdcde6c0baa2c#diff-fa2ed50630bc940bd81d9b671f504b6966aa20f55b74994fe44ab63e333d14daL91
https://github.com/ThaPwned/WCS/pull/34/commits/474a6f74e2fcf110ee0ad3e037ccdcde6c0baa2c#diff-fa2ed50630bc940bd81d9b671f504b6966aa20f55b74994fe44ab63e333d14daL68
https://github.com/ThaPwned/WCS/pull/34/commits/474a6f74e2fcf110ee0ad3e037ccdcde6c0baa2c#diff-fa2ed50630bc940bd81d9b671f504b6966aa20f55b74994fe44ab63e333d14daL57
https://github.com/ThaPwned/WCS/pull/34/commits/474a6f74e2fcf110ee0ad3e037ccdcde6c0baa2c#diff-fa2ed50630bc940bd81d9b671f504b6966aa20f55b74994fe44ab63e333d14daL54
https://github.com/ThaPwned/WCS/pull/34/commits/474a6f74e2fcf110ee0ad3e037ccdcde6c0baa2c#diff-fa2ed50630bc940bd81d9b671f504b6966aa20f55b74994fe44ab63e333d14daL51
https://github.com/ThaPwned/WCS/pull/34/commits/474a6f74e2fcf110ee0ad3e037ccdcde6c0baa2c#diff-fa2ed50630bc940bd81d9b671f504b6966aa20f55b74994fe44ab63e333d14daL48

**Comment by srpg** [comment]
And here some more
https://github.com/ThaPwned/WCS/pull/34/commits/474a6f74e2fcf110ee0ad3e037ccdcde6c0baa2c#diff-fa2ed50630bc940bd81d9b671f504b6966aa20f55b74994fe44ab63e333d14daL45

https://github.com/ThaPwned/WCS/pull/34/commits/474a6f74e2fcf110ee0ad3e037ccdcde6c0baa2c#diff-fa2ed50630bc940bd81d9b671f504b6966aa20f55b74994fe44ab63e333d14daL42

https://github.com/ThaPwned/WCS/pull/34/commits/474a6f74e2fcf110ee0ad3e037ccdcde6c0baa2c#diff-fa2ed50630bc940bd81d9b671f504b6966aa20f55b74994fe44ab63e333d14daL39

File: resource/source-python/translations/wcs/chat_strings.ini [hunk 1]
```diff
@@ -2,277 +2,352 @@
...
 [no unused]
     en = "{lightgreen}No unused points."
     ru = "{lightgreen}Нет неиспользованных очков."
     fi = "{lightgreen}Ei käyttämättömiä pisteitä."
+    da = "{lightgreen}Du har ingen ubrugte skill points."
 [no access]
     en = "{lightgreen}You do not have access."
     ru = "{lightgreen}У Вас нет доступа."
-    fi = "{lightgreen}Sinulla ei ole oikeuksia."
+    fi = "{lightgreen}Sinulla ei ole oikeusia."
```
Comment (srpg) [inline resource/source-python/translations/wcs/chat_strings.ini:138]: It should be sinulla ei ole oikeuksia

File: resource/source-python/translations/wcs/chat_strings.ini [hunk 2]
```diff
@@ -2,277 +2,352 @@
...
     fi = "{green}{name} {lightgreen}on vaihdettu race {green}{value}{lightgreen}."
+    da = "{green}{name} {lightgreen}skiftede race til {green}{value}{lightgreen}."
 [admin changerace self]
     en = "{lightgreen}You've changed your race to {green}{value}{lightgreen}."
     ru = "{lightgreen}Вы сменили расу на {green}{value}{lightgreen}."
     fi = "{lightgreen}Sinä vaihdoit sinun racesi {green}{value}{lightgreen}."
+    da = "{lightgreen}Du skiftede din race til {green}{value}{lightgreen}."
 [admin gain bank levels all]
     en = "{lightgreen}You have given all online players {green}{value} level(s){lightgreen} to their bank."
-    fi = "{lightgreen}Sinä olet antanut kaikkille paikalla oleville pelaajille {green}{value} levelei{lightgreen} heidän pankkiin."
```
Comment (srpg) [inline resource/source-python/translations/wcs/chat_strings.ini:225]: This need add back

File: resource/source-python/translations/wcs/chat_strings.ini [hunk 3]
```diff
@@ -2,277 +2,352 @@
...
+    da = "{green}{name}{lightgreen} tilføjede {green}{value} level(s) {lightgreen}til din wcsbank konto."
 [admin gain bank levels sender]
     en = "{green}{name} {lightgreen}have gained {green}{value} level(s){lightgreen} to their bank from you."
-    fi = "{green}{name} {lightgreen}on saannut {green}{value} levelei{lightgreen} heidän pankkiin sinulta."
+    ru = ""
+    fi = ""
+    da = "Du tilføjede {green}{value} level(s){lightgreen} til {green}{name} {lightgreen} wcsbank konto."
 [admin gain bank levels self]
     en = "{lightgreen}You have given yourself {green}{value} level(s){lightgreen} to your bank."
-    fi = "{lightgreen}Sinä annoit itsellesi {green}{value} levelei{lightgreen} sinun pankkiin."
```
Comment (srpg) [inline resource/source-python/translations/wcs/chat_strings.ini:234]: Same here need add back

File: resource/source-python/translations/wcs/chat_strings.ini [hunk 4]
```diff
@@ -2,277 +2,352 @@
...
+    da = "{lightgreen}Du skiftede din race til {green}{value}{lightgreen}."
 [admin gain bank levels all]
     en = "{lightgreen}You have given all online players {green}{value} level(s){lightgreen} to their bank."
-    fi = "{lightgreen}Sinä olet antanut kaikkille paikalla oleville pelaajille {green}{value} levelei{lightgreen} heidän pankkiin."
+    ru = ""
+    fi = ""
+    da = "{lightgreen}Du tilføjede{green}{value} level(s){lightgreen} til alle online spilleres wcsbank konti."
 [admin gain bank levels receiver]
     en = "{lightgreen}You've gained {green}{value} level(s) {lightgreen}to your bank from {green}{name}{lightgreen}."
-    fi = "{lightgreen}Sinä sait {green}{value} levelei {lightgreen}sinun pankkiin {green}{name}:ltä{lightgreen}."
```
Comment (srpg) [inline resource/source-python/translations/wcs/chat_strings.ini:228]: This need add back

Comment (ManifestManah) [reply resource/source-python/translations/wcs/chat_strings.ini:228]: Hi srpg!
Thanks for letting me know, I'm a bit of a newbie when it comes to GitHub, I will take the strings you've posted in here and include them in my strings and then I'll update it again, I never meant to overwrite your work (^.^)

File: resource/source-python/translations/wcs/chat_strings.ini [hunk 5]
```diff
@@ -2,277 +2,352 @@
...
 [gain xp]
     en = "{lightgreen}You have gained {green}{value} XP{lightgreen}."
     ru = "{lightgreen}Вы получили {green}{value} опыта{lightgreen}."
     fi = "{lightgreen}Sinä sait {green}{value} XP{lightgreen}."
+    da = "{lightgreen}Du fik {green}{value} XP{lightgreen}."
 [gain xp killed]
     en = "{lightgreen}You have gained {green}{value} XP{lightgreen} for making a kill."
     ru = "{lightgreen}Вы получили {green}{value} опыта{lightgreen} за убийство."
     fi = "{lightgreen}Sinä sait {green}{value} XP{lightgreen} taposta."
+    da = "{lightgreen}Du fik {green}{value} XP{lightgreen} for at dræbe en modstander.""
```
Comment (srpg) [inline resource/source-python/translations/wcs/chat_strings.ini:49]: There is "" it only need be " at end of line

Comment (ManifestManah) [reply resource/source-python/translations/wcs/chat_strings.ini]: Thank you srpg, it should now all be fixed and included in the pull request (^.^)

**Comment by srpg** [comment]
I still saw 3 or 4missing translation in your branch

File: resource/source-python/translations/wcs/chat_strings.ini (hunk 3 above)
Comment (srpg) [reply resource/source-python/translations/wcs/chat_strings.ini:234]: In your branch fi = "", at [admin gain bank levels self]
It should be fi = "{lightgreen}Sinä annoit itsellesi {green}{value} levelei{lightgreen} sinun pankkiin."

**Comment by srpg** [comment]
Quickly looked phone the strings, now all seems to be fine

**Comment by ThaPwned** [comment]
Thanks!

**Pull request merged**